### DISCUSSION TITLE
Data Type Assignment

### DISCUSSION THREAD

**Discussion by ETCappy** [microsoft/Web-Dev-For-Beginners#420] [Q&A] [answered]
Hello- Great Curriculum!  Is there an example of a shopping cart assignment for JS datatypes?
Thanks!

**Comment by jlooper** [comment]
hi, can you link to the assignment here so folks can follow along?

**Reply by ETCappy** [reply]
Hello-
Thanks for replying.  Here is the assignment and I would really like to see the expectation of wha a finished product would look like for this unit of work.
Thanks so much-
[https://github.com/microsoft/Web-Dev-For-Beginners/blob/main/2-js-basics/1-data-types/assignment.md](url)

**Reply by jlooper** [reply]
Here we go https://github.com/crisbnp/fefs-cart

**Reply by ngbeken1monday** [reply]
<a href="https://github.com/crisbnp/fefs-cart">follow this link</a>

**Comment by jlooper** [comment, marked as answer]
https://github.com/crisbnp/fefs-cart

**Reply by sanamhub** [reply]
*GitHub pages for the repo: https://crisbnp.github.io/fefs-cart/

**Comment by sanamhub** [comment]
Also, you can read here https://crisbnp.github.io/fefs-cart/
